nipype/pipeline/engine/utils.py

```diff
@@ -291,7 +291,7 @@ def load_resultfile(results_file, resolve=True):
         raise FileNotFoundError(results_file)
 
     result = loadpkl(results_file)
-    if resolve and result.outputs:
+    if resolve and hasattr(result,"outputs") and result.outputs:
```

Comment:
Simpler and has a space after the commas:

```suggestion
    if resolve and getattr(result, "outputs", None):
```